checker/jdk/nullness/src/javax/xml/parsers/DocumentBuilder.java

```diff
@@ -85,14 +85,14 @@ protected DocumentBuilder () {
      *
      * @since 1.5
      */
-    @SuppressWarnings("nullness")
+    // Commented lines from nullness checker analysis. These dereferences might cause NullPointerExceptions while dereferencing. 
```

Comment:
Please don't write comments like this.  It is better to suppress warnings than to introduce code changes.
Commenting is only appropriate when the code does not compile with the regular type-checker.
Any time you suppress warnings or comment out code, add an explanation of why.
Your explanation above suggests that this is buggy code in the JDK.  Should you report it?
I'm confused how `this.getClass().getName()` can throw a null pointer exception; can you explain?

Reply:
this.getClass().getName() will not cause any problem, it is wrongly commented out. But this.getClass().getPackage().getSpecificationTitle() & this.getClass().getPackage().getSpecificationVersion() can cause NullPointerExceptions as getPackage() does return null. But, I think I'll go with your suggestion and make changes accordingly.
Thanks.